.github/workflows/main-dockerhub.yml

```diff
@@ -1,15 +1,14 @@
 name: dockerhub-main
 
-
 on:
   push:
-    branches: [ master ]
+    branches: [ ILCD-54 ]
```

Comment:
@mati007thm soll das so mit dem branch dirn sein? ich denk das ist nur zum testen so richtig, oder?

Reply:
ohne das läuft das ganze in einen error